### DISCUSSION TITLE
Help Working with WebXR within Enable3D

### DISCUSSION THREAD

**Discussion by Isomonoid** [enable3d/enable3d#168] [Help / Questions]
Hi, I am building a project with Enable3D that is meant to be usable in VR. I have noticed that currently there is no documentation nor examples for how to use Enable3D alongside WebXR in particular on the website.

I have tried to understand how the integration with WebXR might have been designed for use through having a look at the source code, but with limited success (noticing functions such as  `updateXR` and the option `enableXR` but without any concrete examples pointing out to how these might have been intended to be used).

Is the integration with WebXR ready to be used? If possible, I would be happy to try and contribute some documentation for working with WebXR within Enable3D if someone could offer a minimal explanation or pointer with regards to initializing a scene with working VR integration.

Thank  you.

**Comment by yandeu** [comment]
Yes, it is ready and you can use it. See this example: https://enable3d.io/examples/virtual-reality.html.

With enable3d you can do anything that three.js can do.

True, there is no documentation yet, but the XR wrapper around three.js is very small. Most of the things you need is availabe in the three.js docs.

__Do you use phaser or the standalone enable3d framework?_
